sklearn/model_selection/_search.py

```diff
@@ -649,8 +649,14 @@ def fit(self, X, y=None, groups=None, **fit_params):
 
         X, y, groups = indexable(X, y, groups)
         # make sure fit_params are sliceable
-        fit_params_values = indexable(*fit_params.values())
-        fit_params = dict(zip(fit_params.keys(), fit_params_values))
+        # TODO: remove in 0.24
+        try:
+            fit_params_values = indexable(*fit_params.values())
+            fit_params = dict(zip(fit_params.keys(), fit_params_values))
+        except TypeError:
```

Comment:
This means that presence of any scalar men's that indexables will not be processed. I find that a bit surprising.